src/DetailsView/Styles/detailsview.scss

```diff
@@ -436,14 +436,6 @@ div.insights-file-issue-details-dialog-container {
         padding-right: 0px;
         flex-grow: 1;
         min-height: 0;
-        .details-view-nav-pivots {
```

Comment:
dead styles.

Reply:
Also note that the max-height is being calculated incorrectly; that's how I picked up that it is not being used.

(The variable ```detailsViewNavPivotsHeight``` is not being referenced correctly)